src/sensors/softfusion/softfusionsensor.h

```diff
@@ -23,7 +23,12 @@
 
 #pragma once
 
+#include <array>
+#include <type_traits>
+#include <utility>
+
 #include "../SensorFusionRestDetect.h"
+#include "../nonblockingcalibration/NonBlockingCalibration.h"
```

Comment:
This is subjective but I'm not the fan of "nonblocking" name for that. I think something like "auto" or "self-learning" is more descriptive for what it is.